common/coefficients_spectral_response_correction.inc

```diff
@@ -297,8 +297,8 @@
   ! a=6: n channels
   ! b=2: slope/offset
   ! c=2: day/night_tw
-  ! d=22: satellite
-  real, DIMENSION(6,2,2,22) :: n19mimic = reshape( &
+  ! d=28: satellite
+  real, DIMENSION(6,2,2,28) :: n19mimic = reshape( &
```

Comment:
Could you add a brief docstring (or whatever they're called in FORTRAN world!) describing how these values are calculated? Doesn't need to be in huge detail, just something we can use as a quick reference if we need to know in the future.

Reply:
Will do that :)

Reply:
Resolved with b86c2c6